README.md

```diff
@@ -103,6 +103,9 @@ After successfully executing the command above, set the value of environment var
 If you do not do it, the request for VRF using Prepayment won't be working properly.
 After setting the `ACC_ID` in `.env` file, you can move to the next step **Request VRF with Prepayment**.
 
+If you'd like to use your existing prepayment account, set your account id in the `.env` file and add your deployed consumer to your account by executing the hardhat `removeConsumer` task (described in more detail below). Note that providing `accountId` parameter is optional if your account id is already in the `.env` file and `consumer` parameter is also optional if you've deployed your consumer using hardhat under this repository.
```

Comment:
So if I'm not missing something, maybe the command should be addConsumer? 
```suggestion
If you'd like to use your existing prepayment account, set your account id in the `.env` file and add your deployed consumer to your account by executing the hardhat `addConsumer` task (described in more detail below). Note that providing `accountId` parameter is optional if your account id is already in the `.env` file and `consumer` parameter is also optional if you've deployed your consumer using hardhat under this repository.
```

Reply:
Yes, you are right Nick. The actual command is provided near the bottom of the readme: 
```shell
npx hardhat addConsumer \
    --consumer $CONSUMERADDRESS \
    --account-id $ACCOUNT \
    --network $NETWORK
```
with consumer and account-id parameters being optional. I didn't want to repeat the same lines so just mentioned to scroll down. Would you instead recommend providing the command there?

Reply:
Maybe my comment wasn't clear, I was talking about replacing `removeConsumer` into `addConsumer` in the description. I don't think it is necessary to provide full command codes. 🙏

Reply:
oh i see, makes sense, missed that. Will push a quick fix, thanks!